src/common/config.h

```diff
@@ -156,8 +156,13 @@ struct md_config_t {
   void show_config(Formatter *f);
 
   /// obtain a diff between our config values and another md_config_t values
-  void diff(const md_config_t *other,
-            map<string,pair<string,string> > *diff, set<string> *unknown);
+  void diff(const md_config_t *other, map<string,pair<string,string> > *diff, 
+            set<string> *unknown);
+
+  //-- obtain a diff between our config values and another md_config_t values
```

Comment:
could you use `///` to start the comment for a function or method. `///` is used by doxygen to start a short description for a function.